src/com/clover/util/ImageUtil.ts

```diff
@@ -12,8 +12,8 @@ export class ImageUtil implements IImageUtil {
     public getBase64Image(img: HTMLImageElement): string {
         // Create an empty canvas element
         var canvas = document.createElement("canvas");
-        canvas.width = img.width;
-        canvas.height = img.height;
+        canvas.width = img.naturalWidth;
+        canvas.height = img.naturalHeight;
 
```

Comment:
I think we want to be using natureWidth/Height, it will prevent the image from potentially being cutoff if the dimensions of the image are larger than the canvas.

Reply:
👍